kuma/wiki/inline_examples.py

```diff
@@ -0,0 +1,237 @@
+INLINE_EXAMPLES = {
+    "FOR_EACH": """
```

Comment:
Here's ForEach (left is inlined, right is iframed, after hitting "Run"). Only diff appears to be fonts, 👍 

![screenshot 2018-11-02 10 32 59 2](https://user-images.githubusercontent.com/286017/47924859-c3679a00-de8a-11e8-9cd6-eb2c171ed23b.png)